.gitignore

```diff
@@ -42,3 +42,4 @@ admin/.openapi-generator
 scripts/gh-md-toc
 docs/.openapi-generator/*
 
+test/generated/
```

Comment:
Target folder of generated contract. We do not want contract to be part of the repository. It is used only by CI/CD